tests/Integration/UI/SettingsPageTest.php

```diff
@@ -44,153 +44,185 @@ public function set_up(): void {
 		parent::set_up();
 		self::$parsely       = new Parsely();
 		self::$settings_page = new Settings_Page( self::$parsely );
+
+		add_filter( 'pre_http_request', array( $this, 'mock_request_api_credentials_validation_success' ), 10, 3 );
+	}
+
+	/**
+	 * Tear down method called after each test.
+	 */
+	public function tear_down(): void {
+		parent::tear_down();
+		remove_filter( 'pre_http_request', array( $this, 'mock_request_api_credentials_validation_success' ), 10 );
 	}
 
 	/**
-	 * Verifies that valid Site ID values are retained when validated.
+	 * Verifies that empty Site ID and API Secret values are retained when validated.
 	 *
-	 * @since 3.9.0
+	 * @since 3.11.0
 	 *
 	 * @covers \Parsely\UI\Settings_Page::validate_basic_section
-	 * @uses \Parsely\Parsely::__construct
-	 * @uses \Parsely\Parsely::api_secret_is_set
-	 * @uses \Parsely\Parsely::are_credentials_managed
-	 * @uses \Parsely\Parsely::get_api_secret
-	 * @uses \Parsely\Parsely::get_options
-	 * @uses \Parsely\Parsely::get_managed_credentials
-	 * @uses \Parsely\UI\Settings_Page::__construct
-	 * @uses \Parsely\UI\Settings_Page::get_logo_default
-	 * @uses \Parsely\UI\Settings_Page::get_obfuscated_value
-	 * @uses \Parsely\UI\Settings_Page::get_unobfuscated_value
-	 * @uses \Parsely\UI\Settings_Page::sanitize_site_id
-	 * @uses \Parsely\UI\Settings_Page::validate_advanced_section
-	 * @uses \Parsely\UI\Settings_Page::validate_options
-	 * @uses \Parsely\UI\Settings_Page::validate_options_post_type_tracking
-	 * @uses \Parsely\UI\Settings_Page::validate_recrawl_section
-	 * @uses \Parsely\Validator::validate_site_id
+	 * @uses   \Parsely\Parsely::__construct
+	 * @uses   \Parsely\Parsely::api_secret_is_set
+	 * @uses   \Parsely\Parsely::are_credentials_managed
+	 * @uses   \Parsely\Parsely::get_api_secret
+	 * @uses   \Parsely\Parsely::get_options
+	 * @uses   \Parsely\Parsely::get_managed_credentials
+	 * @uses   \Parsely\UI\Settings_Page::__construct
+	 * @uses   \Parsely\UI\Settings_Page::get_logo_default
+	 * @uses   \Parsely\UI\Settings_Page::get_obfuscated_value
+	 * @uses   \Parsely\UI\Settings_Page::get_unobfuscated_value
+	 * @uses   \Parsely\UI\Settings_Page::sanitize_site_id
+	 * @uses   \Parsely\UI\Settings_Page::validate_advanced_section
+	 * @uses   \Parsely\UI\Settings_Page::validate_options
+	 * @uses   \Parsely\UI\Settings_Page::validate_options_post_type_tracking
+	 * @uses   \Parsely\UI\Settings_Page::validate_recrawl_section
+	 * @uses   \Parsely\Validator::validate_api_credentials
 	 *
 	 * @group settings-page
 	 * @group settings-page-validation
 	 */
-	public function test_valid_site_id_values_are_retained_when_validated(): void {
+	public function test_empty_site_id_api_secret_are_retained_when_validated(): void {
```

Comment:
```suggestion
	public function test_empty_api_credentials_are_retained_when_validated(): void {
```
I think this is better? If not, feel free to keep as-is.